### DISCUSSION TITLE
7기 대상 홍보 및 운영서버 템플릿 데이터

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#731] [이야기해봐요]
#  결정사항

## 어디에, 어떻게?

- 우아한테크 유투브에 프로젝트 소개 영상 건의하기
- 각자 개인 블로그에 '합격후기' 에 슬쩍 함께 홍보하기
- 7기 커뮤니티에 홍보글 올리기
  - 코드잽을 써보고 피드백, 개선사항 등을 문의사항으로 보내면 스타벅스 기프티콘 등 리워드 드려요! 내용 포함
- 각자 매일 하나씩 유의미한 템플릿 올리기
- 현재 운영서버의 중복 템플릿 모두 삭제하기

## 일정
- 프리코스 1차 미션 종료 후 2차 미션 시작할때

**Comment by Hain-tain** [comment]
- 커뮤니티 들어가서 홍보글 올리면 될까..??
- 6기때 코드 우리가 줘도 되나? (사실 이거 올리면 관심은 조금 더 받을 수 있을 것 같다. 그러나 우테코에서는 프리코스때 최대한 선배 기수꺼 보고 따라하지 말라고 안내하는데 우리가 우테코의 교육 철학에 반하는거 아닐까 걱정이 조금 되긴 한다.)
- 홍보 전에 좋은 글들이 많이 있어야 유의미할 것 같다... 우리라도 좋은 템플릿 몇개씩 올려보자!!

**Reply by Jaymyong66** [reply]
커뮤니티 ok
코드 주면 안될듯
좋아

**Comment by Hain-tain** [comment]
## 어떻게?

**Reply by Hain-tain** [reply]
유투브 프로젝트 소개 영상 올리자!

**Reply by Jaymyong66** [reply]
- 우아한테크 유튜브에 6기 프로젝트 유튜브 영상 기획을 해달라
- 코드잽을 써보고 피드백, 개선사항 등을 문의사항으로 보내면 스타벅스 기프티콘 등을 주자 -> 6기 & 7기
- 개인 홍보 열심히

**Reply by Hain-tain** [reply]
개인 블로그에 '합격후기' 에 슬쩍 홍보 함께하기

**Comment by jminkkk** [comment]
## 홍보 포인트

"관리"

7기 프리코스 2차 미션 때 진행 -> 6기 홍보도 동시 진행

**Reply by jminkkk** [reply]
# 기존 데이터가 빈약하다

매일 데일리 때 각자꺼 적기
